core/src/main/java/com/linecorp/armeria/server/DefaultServiceRequestContext.java

```diff
@@ -112,6 +113,8 @@
     @Nullable
     private String strVal;
 
+    private final EventLoop eventLoop;
```

Comment:
Could you move this field right next to `ch`? We probably should keep this close to the beginning of the memory layout because it's accessed very often.